### ISSUE TITLE
Robot load: GDA should call to Hyperion immediatly after robot load

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#1047]
With the new plan made in https://github.com/DiamondLightSource/hyperion/pull/1038 we should modify GDA to call Hyperion immediately after robot load.

## Acceptance Criteria
* GDA calls Hyperion immediately after robot load
* Once Hyperion is done the logic flow goes back to GDA just after it would have done an xray centre
* There is a property to turn this on/off

**Comment by DominicOram** [comment]
Initial work at https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/40773 proved that we could trigger Hyperion immediately after robot load and GDA was happy

**Comment by DominicOram** [comment]
On a second look at the GDA code, as a first pass it will probably be better to trigger Hyperion straight after robot load + ispyb deposit. This will still give us a speed up in the code but will also be easier as a first step

**Comment by DominicOram** [comment]
See the following changes in gerrit:
* https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/40786
* https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/40789
* https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/40773

**Comment by DominicOram** [comment]
These have all now been merged so closing this. Unfortunately it didn't actually give us much improvement
